src/mcts/search.cc

```diff
@@ -545,7 +551,12 @@ std::vector<EdgeAndNode> Search::GetBestChildrenNoTemperature(Node* parent,
         if (a_rank == kNonTerminal) {
           // Prefer largest playouts then eval then prior.
           if (a.GetN() != b.GetN()) return a.GetN() > b.GetN();
-          if (a.GetWL() != b.GetWL()) return a.GetWL() > b.GetWL();
+          // Default doesn't matter here so long as they are the same as either
```

Comment:
So... a separate question here.  Should this be using GetWL instead of GetQ??  Draw score is being ignored in tiebreaking which seems wrong.

Reply:
I somehow thought that D are equal in that case, but indeed they are not and Q makes more sense then WL.

Reply:
Implemented.